src/components/Homepage/LendByCategory/LoanCategory.vue

```diff
@@ -0,0 +1,526 @@
+<template>
+	<div
+		class="category-row-hover"
+	>
+		<div class="row title-row">
+			<div class="column small-12">
+				<h2 class="category-name">
+					{{ cleanName }}
+				</h2>
+				<p class="category-description show-for-large">
+					{{ loanChannel.description }}
+				</p>
+			</div>
+		</div>
+
+		<div
+			class="cards-and-arrows-wrapper"
+			ref="outerWrapper"
+		>
+			<span
+				class="arrow left-arrow"
+				:class="{inactive: scrollPos === 0}"
+				ref="leftArrow"
+				@click="scrollRowLeft"
+			>&lsaquo;</span>
+			<div class="cards-display-window" ref="innerWrapper">
+				<loading-overlay v-if="isLoading" id="updating-overlay" />
+				<div
+					v-else
+					class="cards-holder"
+					:style="{ marginLeft: scrollPos + 'px' }"
+					v-touch:swipe.left="scrollRowRight"
+					v-touch:swipe.right="scrollRowLeft"
+
+					ref="hoverCardsHolder"
+				>
+					<loan-card-controller
+						loan-card-type="LendHomepageLoanCard"
+						class="is-in-category-row"
+						v-for="(loan, index) in loans"
+						:key="loan.id"
+						:loan="loan"
+						:items-in-basket="itemsInBasket"
+						:category-id="loanChannel.id"
+						:category-set-id="setId"
+						:row-number="rowNumber"
+						:card-number="index + 1"
+						:enable-tracking="true"
+						:is-visitor="!isLoggedIn"
+					/>
+
+					<!--
+						Blocks of attributes above:
+						1) Props for implemented loan cards
+					-->
+					<div v-if="showViewAllLink" class="column column-block is-in-category-row view-all-loans-category">
+						<router-link
+							:to="cleanUrl"
+							:title="`${viewAllLoansCategoryTitle}`"
+							v-kv-track-event="[
+								'Lending',
+								'click-View all',
+								`Loan card`]"
+						>
+							<div
+								class="see-all-card"
+								:class="seeAllCardClass"
+							>
+								<div class="link">
+									{{ viewAllLoansCategoryTitle }}
+								</div>
+							</div>
+						</router-link>
+					</div>
+				</div>
+			</div>
+			<span
+				class="arrow right-arrow"
+				:class="{inactive: scrollPos <= minLeftMargin}"
```

Comment:
rather than a class of inactive, this could be a `disabled` attribute on the button